Automattic/newspack-plugin
# fix: remove pop-ups ui from engagement wizard

### All Submissions:






### Changes proposed in this Pull Request:

Remove the Pop-ups screen and endpoints from the Engagement wizard, now that Pop-ups have their own wizard.

### How to test the changes in this Pull Request:

1. Verify the Pop-ups screen is gone from the Engagement wizard.
2. Verify the Engagement wizard works as it did before.
3. Verify the Pop-ups wizard works as is it did before.

### Other information:
